tests/lean/assumption.lean.expected.out

```diff
@@ -0,0 +1,3 @@
+{"msgs":[{"caption":"trace output","file_name":"assumption.lean","pos_col":2,"pos_line":3,"severity":"information","text":"Try this: h1\n"}],"response":"all_messages"}
```

Comment:
this file is actually garbage, I don't know how to get the output of the test

Reply:
```
cd tests/lean
./test_single.sh ../../bin/lean assumption.lean
```

Reply:
The output is still wrong.  Please fix it until the command above prints `-- checked`.

Reply:
hmm actually I still don't know how to build `bin/lean` in the first place

Reply:
Yes, where are you stuck?

Reply:
now I'm just waiting on the build. before I was stuck on understanding the error messages that came out of `./test_single.sh ../../bin/lean assumption.lean`

Reply:
Now I have a real lean binary. I'm getting "error: unexpected token" at the question mark. Most of my debugging strategies rely on VSCode; is there a way to tell it to use the local lean binary?

Reply:
Hmm. Seems like the current commit gives lots of errors like this. I thought the approach using the optional parse argument would prevent these. I think I should give up.
```
/Users/jstark/lean-projects/lean/library/init/meta/interactive.lean:1057:30: error: type mismatch at application
  try assumption
term
  assumption
has type
  parse (tk "?")? → tactic unit
but is expected to have type
  itactic

Reply:
I think you should just replace these references to `assumption` by `tactic.assumption`.  The `?` parser only works when the tactic is parsed in an `begin ... end` block (or `by ...`).

Reply:
Okay, I think I have implemented that suggestion in the latest commit. I'm still getting the following, which seems to be the opposite problem (i want it to use the version with the parser and it uses the version without) I tried adding `import tactic.interactive` but i could not resolve that import.
```
/Users/jstark/lean-projects/lean/tests/lean/assumption.lean:3:12: error: unexpected token
/Users/jstark/lean-projects/lean/tests/lean/assumption.lean:4:0: error: invalid 'end', there is no open namespace/section